sdrbase/util/aprs.cpp

```diff
@@ -156,9 +176,14 @@ bool APRSPacket::decode(AX25Packet packet)
             }
 
             return true;
+        } else {
+            qDebug() << "APRSPacket::decode: AX.25 Destination did not match known regexp " << m_to;
```

Comment:
I added this as some of the other Features throw out debug messages when there are failures, such as 

> PacketDemodSink::processOneSample: CRC mismatch: 69e7 c228